pkg/users/account-details.js

```diff
@@ -96,40 +96,62 @@ function get_expire(name) {
             .then(parse_expire);
 }
 
-export function AccountDetails({ accounts, groups, current_user, user, shells }) {
+export function AccountNotFound() {
+    return (
+        <EmptyState headingLevel="h1" icon={ExclamationCircleIcon} titleText={_("Account not available or cannot be edited.")} variant={EmptyStateVariant.sm} id="account-failure">
+            <EmptyStateFooter>
+                <EmptyStateActions>
+                    <Breadcrumb>
+                        <BreadcrumbItem to="#/">{_("Back to accounts")}</BreadcrumbItem>
+                    </Breadcrumb>
+                </EmptyStateActions>
+            </EmptyStateFooter>
+        </EmptyState>
```

Comment:
These 11 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-f759ca81-20250825-111524-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)